.travis.yml

```diff
@@ -0,0 +1,26 @@
+language: c
+compiler: gcc
+sudo: required
+dist: trusty
+
+services:
+  - docker
+
+addons:
+  apt:
+    packages:
+    - bash
+    - tar
+    - bzip2
+
+env:
+  global:
+    # The next declaration is the encrypted COVERITY_SCAN_TOKEN, created
+    # via the "travis encrypt" command using the project repo's public key
```

Comment:
Where does this key come from? I checked https://api.travis-ci.org/repos/sssd/sssd/key and I just got not found..

Reply:
On Coverity you have access to a key for the project. Then you call "travis encrypt $key" and that's how I got the key.